internal/model/comment.go

```diff
@@ -0,0 +1,96 @@
+package model
+
+import (
+	"fmt"
+	"time"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+// Comments represents a slice of a comment model.
+type Comments []Comment
+
+// CommentsDTO represents a slice of a dto comment model.
+type CommentsDTO []CommentDTO
+
+// Comment represents a comment model.
+type Comment struct {
+	ID         primitive.ObjectID `bson:"_id,omitempty"`
```

Comment:
Hm... I think it's not the right place for mongo primitives. You should use the mongo package only in your mongo folder